src/index.html

```diff
@@ -4,19 +4,109 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, initial-scale=1.0"
     />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
     />
-    <title>Moyo header</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
     />
+    <title>Document</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="img__container">
+        <a
+          href="#"
+          class="img__link"
+        >
+          <img
+            src="https://github.com/sinner1993/layout_moyo-header/blob/master/src/images/logo.png?raw=true"
+            alt="moyo logo"
+          />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item first__item">
```

Comment:
```suggestion
          <li class="nav__item">
```

Reply:
can you give me example how can i change it?